integration-tests/cli/price-export.it.js

```diff
@@ -0,0 +1,210 @@
+import fs from 'fs'
+import tmp from 'tmp'
+import path from 'path'
+import csv from 'csvtojson'
+import { getCredentials } from '@commercetools/get-credentials'
+import { exec } from 'child_process'
+import {
+  sampleProductType,
+  sampleCustomerGroup,
+  sampleChannel,
+  sampleCustomType,
+  createProducts,
+} from './helpers/prices.data'
+import { createData, clearData } from './helpers/utils'
+import { version } from '../../packages/price-exporter/package.json'
+
+let projectKey
+if (process.env.CI === 'true')
+  projectKey = 'price-export-integration-test'
+else
+  projectKey = process.env.npm_config_projectkey
+
+describe('Price Exporter', () => {
+  let apiConfig
+  const productType = { typeId: 'product-type' }
+  const customerGroup = { typeId: 'customer-group' }
+  const channel = { typeId: 'channel' }
+  const customType = { typeId: 'type' }
+  const bin = './integration-tests/node_modules/.bin/price-exporter'
+
+  beforeAll(() => getCredentials(projectKey)
+    .then((credentials) => {
+      apiConfig = {
+        host: 'https://auth.sphere.io',
+        apiUrl: 'https://api.sphere.io',
+        projectKey,
+        credentials,
+      }
+      return clearData(apiConfig, 'products')
+    })
+    .then(() => clearData(apiConfig, 'types'))
+    .then(() => clearData(apiConfig, 'channels'))
+    .then(() => clearData(apiConfig, 'customerGroups'))
+    .then(() => clearData(apiConfig, 'productTypes'))
+    .then(() => createData(apiConfig, 'productTypes', [sampleProductType]))
+    .then((data) => {
```

Comment:
@junajan Only in before is also a problem: `price-exporter` could not clear data before because `inventory-export` did not clear the inventory when done and `customTypes`/`channels` cannot be cleared whiled linked to an inventory... Had to refactor that part of the int test